<script>
    async function myfunc(){

        try{
            var res1 = await fetch('https://fakestoreapi.com/products/category/electronics');
            var res2 = await fetch('https://fakestoreapi.com/products/category/jewelery');
            var res3 = await fetch('https://fakestoreapi.com/products');
        
            var ans1 = await res1.json();
            var ans2 = await res2.json();
            var ans3 = await res3.json();
            return [ans1,ans2,ans3]
        }
        catch(err){
            // console.log(err);
            return {msg:"Api Fail",status:404}
        }
        
        // console.log(ans1);
        // console.log(ans2);
        // console.log(ans3);

        
    }

    var response = myfunc()
    console.log(response);

    response.then(ans=>{
        console.log(ans);
    }).catch(err=>{
        console.log(err);
    })
</script>